responsible-ai-collaborative/aiid
# Fix canonical urls

Resolves #1349. Also adds canonical urls to pages where they were missing.

## Breakage Attempts / Testing

- In Spanish, check that the canonical url is correct for every page in the sidebar
  - **Found issue**: Still missing from /taxonomies. Fixed.
  - **Found issue**: Still missing from /blog, along with Helmet generally. Fixed.
- Check /taxonomies/cset
  - **Found issue**: Whole helmet is still missing. Fixed.
- Check /entities/facebook
  - **Found issue**: Whole helmet is still missing. Fixed.
- Check every page linked in the footer
  - **Found issue**: canonical url for `research/related-work` is `research/4-related-word`. Changed url to include number, matching the others research pages.
